lib/rules/max-statements-per-line.js

```diff
@@ -35,11 +35,12 @@ module.exports = {
         const sourceCode = context.getSourceCode(),
             options = context.options[0] || {},
             maxStatementsPerLine = typeof options.max !== "undefined" ? options.max : 1,
-            message = "This line has too many statements. Maximum allowed is {{maxStatementsPerLine}}.",
-            data = { maxStatementsPerLine };
+            emptyBlockMessage = "This line has an empty block, but the maximum allowed is 0.",
```

Comment:
The comment farther down in the file suggests it's for backwards compatibility.

You're right though that this message could be made clearer - it just didn't seem like "This line has N statements" made sense. Any suggestions?